### DISCUSSION TITLE
美食推荐

### DISCUSSION THREAD

**Discussion by TTGFCCt** [share121/inter-knot#2123] [其他]
![20240814204804](https://github.com/user-attachments/assets/0fe09354-fc0e-47b5-9720-1a93c1c9c60c)
家人们(*´∀`*)今天在六分街遇到了宝藏级美食店♡´･ᴗ･`♡真的好好吃！！！
店铺老板@在后面了
@乔普师傅

**Comment by Loring121** [comment]
这个委托我，不是，
这家店我探定了！

**Comment by ENE4546** [comment]
这家我吃过！真的好吃到跺jiojio,我直接风暴吸入
![430fef53-4b10-46f5-9430-53417c02ee96](https://github.com/user-attachments/assets/22f9c5e9-f8a5-43aa-8407-c0fb4121f4a6)

**Comment by Wangsheng-Zhongli** [comment]
六分街的美食，果然名不虚传。既然被称为“宝藏级”，想必其中必有独到之处。若有机会，倒是愿意前去品尝一番，见识这位乔普师傅的手艺，也不失为一次趣味横生的体验。

**Comment by GWWA** [comment]
是录像店旁边的那一家吗，我去租录像时经常去他家吃饭，确实不错！推荐！

**Comment by blblle** [comment]
嗯嗯呢(超好吃的),嗯呢呢嗯呢呢呢(法厄同爱吃),嗯呢嗯呢呢(价格有点贵)

**Comment by Gobbo618** [comment]
这家店爆赞！老板人超好的（PS：强推加双份冰的冰汤面！！！）
